commands/add/__tests__/add-command.test.js

```diff
@@ -88,13 +87,29 @@ describe("AddCommand", () => {
   });
 
   it("should reference to devDepdendencies", async () => {
-    const testDir = await initFixture("basic");
```

Comment:
Separate test cases, please. One initFixture per test case, just like all the others.

After the existing initial dev test, you only need to make assertions on one package.